src/_includes/layouts/membership.liquid

```diff
@@ -1,3 +1,30 @@
 ---
 layout: layouts/base.liquid
 ---
+<div class="bg-parallax-photo">
+  <div
+    :speed-factor="0.8"
+    sectionClass="parallax-outside"
+    containerClass="parallax-inside">
+    <img src="/assets/img/bg-triangle-pink3.png" class="float-right" />
+  </div>
+</div>
+<div class="container py-5">
+  <div class="row justify-content-md-between">
+    <div class="col-md-6 py-md-5">
+      <h1 class="text-magenta my-md-5">{{ membershipHeader }}</h1>
+      {% if membershipAnnouncement != "" %}
+        {{ membershipAnnouncement | markdownify }}
```

Comment:
You need the `a` tags that link to the membership policy form and application.

Reply:
`                        <b>We are now accepting applications for membership in the DSC from April 10th through May 7th, 2023.<br/><br/>Prior to filling out an application please review the 2023 DSC Membership Policy to ensure your company is eligible and to see the cost of membership: </b><b><a href="/pdfs/Membership-Policy-2023.pdf" target="_blank">DSC 2023 Membership Policy</a></b><br/><br/>
                        <b>Fill out your application here: <a href="https://www.surveymonkey.com/r/WQ5JRKR">DSC 2023 Membership Application</a></b>`

Reply:
I added the announcement text through the CMS, links should be fixed in [13780f5](https://github.com/kindsys/digital-services-coalition/pull/23/commits/13780f51a05ab2c88e2c293a51ea03279538ef35)